sky/backends/cloud_vm_ray_backend.py

```diff
@@ -2938,6 +2953,7 @@ def _execute_task_n_nodes(self, handle: ResourceHandle, task: task_lib.Task,
         if is_tpu_vm_pod:
             num_actual_nodes = tpu_utils.get_num_tpu_devices(
                 handle.launched_resources)
+            num_actual_nodes = task.num_nodes
```

Comment:
Do we need to keep the two lines right before this?

Reply:
My bad. This line is not needed. Removed